Portal/vip-visualization/src/main/java/fr/insalyon/creatis/vip/visualization/client/view/AbstractViewTab.java

```diff
@@ -62,20 +59,22 @@ public void load() {
 
     private void loadLFN(String lfn) {
         VisualizationServiceAsync vs = VisualizationService.Util.getInstance();
-        modal.show("Loading data file...", true);
+        Layout.getInstance().setNoticeMessage(
```

Comment:
Shouldn't this message disappear (be hidden at some point) ?

Reply:
This message is replaced by the next message "Loading and parsing image …".
This latter message is then removed by code in the javascript file ami-viewer.js "window.hideMessage()" in case of success, or by "window.setWarningMessage('Error loading ' + url + '<br />' + error)", in case an error is detected.

The messages only stay if there an internal AMI error that does not bubble up to our javascript code.